events/README.md

```diff
@@ -0,0 +1,416 @@
+    "template_id": "<template_id that you wish to associate w/ the room>",
+    "region": "in"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<id>",
+        "name": "<room_name>",
+        "enabled": true,
+        "description": "This is a sample description for the room",
+        "customer": "<customer_id>",
+        "recording_info": {
+          "enabled": false
+        },
+        "template_id": "<template_id>",
+        "region": "in",
+        "created_at": "",
+        "updated_at": ""
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't create room. Please try again later'`
+
+## GET /rooms
+
+Get all the created rooms.
+
+- Params:
+  - enabled: true/false
+  - hits: Number of rooms in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<ROOM_OBJECT>,
+      		<ROOM_OBJECT>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get rooms. Please try again later'`
+
+## POST - /join
+
+Generate an auth token for a peer to join a room.
+
+To join the event (for guests, host, maven, moderator)
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Required: **`room_id=[string]`** (The ID of the room to which the user belongs)
+  - Required: **`user_id=[string]`** (The ID of the user for whom the token is being generated)
+  - Required: **`role=[string]`** (The role of the user in the room)
+  ```json
+  {
+    "room_id": "<room_id>",
+    "user_id": "<user_id>",
+    "role": "<role>"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "token": "<auth_token>",
+        "msg": "Token generated successfully!",
+        "success": true
+      }
+      ```
+- **Error Response:**
+  - **Code:** 500
+    - **Content**
+      **`{ "msg": "Some error occured!", "success": false }`**
+
```

Comment:
does this route not require authentication. as cannot see 401 error response.

Reply:
Anyone can send request to join the live session. So, didn't add the authentication error.

Reply:
Yes, I have removed it.